### PR TITLE
Edit and update the release and tagging process in README.

### PR DESCRIPTION
**Pull request by nixor**
Diego and I have tested that this is the proper way to trigger a release. Github bot will handle the release automatically once the MR is tagged correctly.

### CODE REVIEW

File: README.md
Comment (Thilo-Acrolinx) [inline README.md:190]: We had a similar discussion last week with Abhijeet when releasing the sidebar-sdk-java. I think there is a small flaw, since GitHub would build this twice, one when you create the new MR and one more time when you merge it to the main branch. Not sure what would happen when the same release assets are created and uploaded twice?

For the sidebar-sdk-jav we therefore did it like this:
- create an MR where we updated the currentVersion property
- GitHub builds this branch and releases the artifacts
- If all went well, close the MR and discard the branch without merging
- then create a new MR and update the currentVersion to "x.y.z-SNAPSHOT" (whatever the new snapshot version after the release should be). Get an approval and merge to main.

Maybe we should do the same here for this repo?

Comment (nixor) [reply README.md:190]: Hi Thilo we can handle it as you suggest if it works. I made the proposal just to update the way we tested that it works currently as we don’t use Travis anymore.
We can just close this PR and you can update it the way you think it works best. I have no objection there.

Comment (willynganga) [reply README.md:190]: @Thilo-Acrolinx is right. We have been thinking of how to publish the artifacts only once. 
Also, this looks like a good feature @nixor, maybe we can re-open the MR once we figure out a practical way of publishing the artifacts only once.

To clarify, who owns the GitHub Bot (is it GitHub)? Or this is a custom GitHub bot that you guys created.

Comment (nixor) [reply README.md:190]: @willynganga @Thilo-Acrolinx  Somebody created a while back.

In the case that you plan to improve on the current process, sure you can reopen another PR to fix this in the proper way you deem fit.

But as it stand I felt the need to update the read me. At least for the moment this is a way that it works. As we no longer use Travis anymore.

Therefore i think we should approve at least this readme edit changes. And when you have time to look into the future and decide how it should be properly done, then we can make another change.

File: README.md
Comment (abhijeetnarvekar) [inline README.md:182]: @nixor 

issue: Drop the SNAPSHOT while creating a release

Steps for releasing:

- Create a task branch.
- Drop the SNAPSHOT from gradle.properties file. example `0.11.1`
- Merge PR to main
- Create a tag via using the following commands (Or the GitHub UI)
  ```bash
  git tag v[major].[minor].[fix] (etc. v0.11.0)
  git push origin tag v[major].[minor].[fix]
  ```
- Github will create the specified tag on the remote repository and it will build and make a release when merged with the main branch there. Artifacts are automatically generated.

Once tag is created:

- Created new task branch
- Add new SNAPSHOT version. example: `0.12.0-SNAPSHOT`
- Create a PR and merge to main

**Review by abhijeetnarvekar — CHANGES_REQUESTED** [review]

File: README.md
Comment (nixor) [reply README.md]: @abhijeetnarvekar Thanks adjusted.

**Review by abhijeetnarvekar — APPROVED** [review]

**Pull request merged**
